technologies/app/jupyspark/jupyter-spark/resources/tests/test.ipynb.txt

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
Is this file used during build process ?

Reply:
nop ... will be removes only here to facilitate basic testing